### ISSUE TITLE
Config Error Loop

### ISSUE THREAD

**Issue by GavvyDizzle** [quantiom/AdvancedVanish#6]
I just downloaded the newest version (1.1.4) and put it in my /plugins folder. After loading my server for the first time with it in there, the config continuously makes backups until it causes a StackOverflowError. I am using Paper 118.2 if it matters.

```[03:52:53] [Server thread/INFO]: [AdvancedVanish] Enabling AdvancedVanish v1.1.4
[03:52:53] [Server thread/INFO]: [AdvancedVanish] [ACF] Enabled Asynchronous Tab Completion Support!
[03:52:53] [Server thread/WARN]: [AdvancedVanish] WARNING: Config version mismatch, a backup of the old config has been made and the current config has been replaced.
[03:52:53] [Server thread/WARN]: [AdvancedVanish] WARNING: Config version mismatch, a backup of the old config has been made and the current config has been replaced.
... outputs many times
[03:52:58] [Server thread/ERROR]: Error occurred while enabling AdvancedVanish v1.1.4 (Is it up to date?)
java.lang.StackOverflowError: null
	at org.apache.logging.log4j.message.ParameterizedNoReferenceMessageFactory.newMessage(ParameterizedNoReferenceMessageFactory.java:105) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.message.AbstractMessageFactory.newMessage(AbstractMessageFactory.java:99) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logMessage(AbstractLogger.java:2057) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logIfEnabled(AbstractLogger.java:1931) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.warn(AbstractLogger.java:2805) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.handleLogMessageException(AbstractLogger.java:2225) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.tryLogMessage(AbstractLogger.java:2208) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logMessageTrackRecursion(AbstractLogger.java:2159) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logMessageSafely(AbstractLogger.java:2142) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logMessage(AbstractLogger.java:2017) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.logIfEnabled(AbstractLogger.java:1983) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.apache.logging.log4j.spi.AbstractLogger.warn(AbstractLogger.java:2671) ~[log4j-api-2.17.1.jar:2.17.1]
	at org.bukkit.craftbukkit.v1_18_R2.util.ForwardLogHandler.publish(ForwardLogHandler.java:34) ~[paper-1.18.2.jar:git-Paper-386]
	at java.util.logging.Logger.log(Logger.java:980) ~[?:?]
	at java.util.logging.Logger.doLog(Logger.java:1007) ~[?:?]
	at java.util.logging.Logger.log(Logger.java:1030) ~[?:?]
	at me.quantiom.advancedvanish.AdvancedVanish$Companion.log(AdvancedVanish.kt:20) ~[AdvancedVanish.jar:?]
	at me.quantiom.advancedvanish.config.Config.reload(Config.kt:60) ~[AdvancedVanish.jar:?]
        ... outputs this until the plugin disables

**Comment by quantiom** [comment]
Hi, could you please provide your old config file so I can test it out? Thanks.

**Comment by quantiom** [comment]
Fixed in v1.1.5 (3ffe026adbe4bef81f09f018931d0cdda1f52619)
